tillerinobot/src/main/java/tillerino/tillerinobot/UserDataManager.java

```diff
@@ -224,6 +224,14 @@ public void setShowWelcomeMessage(boolean welcomeMessage) {
 			changed |= welcomeMessage != this.showWelcomeMessage;
 			this.showWelcomeMessage = welcomeMessage;
 		}
+
+		@Getter
+		boolean doOsuTrackUpdateOnWelcome = false;
```

Comment:
`osuTrackWelcomeEnabled`. That way the getter is `isOsuTrackWelcomeEnabled` and the setter is `setOsuTrackWelcomeEnabled`. Sounds much more natural.